### PR TITLE
[onert] Include builtin train tensor registry in trainable executor

### PR DESCRIPTION
**Pull request by jyoungyun**
This commit includes builtin train tensor registry in trainable executor.

ONE-DCO-1.0-Signed-off-by: Jiyoung Yun <jy910.yun@samsung.com>

### CODE REVIEW

**Review by ragmani — APPROVED** [review]
LGTM

File: runtime/onert/core/src/compiler/ExecutorFactory.cc [hunk 1]
```diff
@@ -726,7 +726,7 @@ exec::IExecutor *ExecutorFactory::createTrainableExecutor(
   }
   base_backend_contexts.clear();
 
-  train::TensorRegistries tensor_regs{tbackend_contexts, false};
+  train::TensorRegistries tensor_regs{tbackend_contexts, true};
 
   initializeSubgraphIOTensors(
     *lowered_graph, tbackend_contexts,
```
Comment (ragmani) [inline runtime/onert/core/src/compiler/ExecutorFactory.cc:732]: Could you also remove calling getOutputs() like below?
```suggestion
    *lowered_graph, tbackend_contexts,
    lowered_graph->graph().getInputs() | ir::Remove::DUPLICATED | ir::Remove::UNDEFINED);
```

Comment (ragmani) [inline runtime/onert/core/src/compiler/ExecutorFactory.cc:732]: Could you remove calling `getOutputs()` like below
```suggestion
    *lowered_graph, tbackend_contexts,
    lowered_graph->graph().getInputs() | ir::Remove::DUPLICATED | ir::Remove::UNDEFINED);
```

**Comment by jyoungyun** [comment]
@nnfw-bot test onert-x64-release

**Review by glistening — APPROVED** [review]

**Pull request merged**